### ISSUE TITLE
How to Overcome Perfectionism According to Research

### ISSUE THREAD

**Issue by farosato** [farosato/fabiorosatodotcom#9]
---
slug: perfectionism
---
Perfectionism can kill you.

Yep, you read that right.

And I'm not making that up. [Science says so](https://pubmed.ncbi.nlm.nih.gov/19383652/).

Now that I got your attention, let's see how to overcome perfectionism once and for all.

Let's start with a very sharp definition of perfectionism, straight from the book ["The Gitfs of Imperfection"](https://brenebrown.com/book/the-gifts-of-imperfection/) by social researcher Brené Brown:

> "Perfectionism is not the same thing as striving to do your best.
> 
> Perfectionism is the belief that if we live perfect, look perfect, and act perfect, we can minimize or avoid the pain of blame, judgment, and shame.
> 
> It's a shield. It's a 20-ton shield that we lug around thinking it will protect us when – in fact – it's the thing that's really preventing us from flight."

That hit hard, didn't it?!

**Perfectionism is not the way. _It's in the way_.**

Ready to change? Great!

Here are the two secrets to overcoming perfectionism according to research 👇

## 1. Reevaluate Your Standards

Take a look around.
How do other people do it?

Look at both the RESULTS and the **COSTS** involved.

How long do they take to complete similar tasks?

Are they still getting acceptable results?

Is what you're doing really necessary?

Most perfectionists will struggle with this. _“Of course everything I do is necessary”_ 🤡 

So really look at the **COSTS involved** — not merely results.

Perfectionists produce good results but they often act like time and driving other people crazy don’t matter.

Is the extra hour you spend reviewing your work really worth finding that _one_ grammatical error?

When ALL the factors are taken into consideration — time, stress, having a life, not making others fantasize about your demise — **are your standards REALISTIC?**

Ask a friend.

If you're finding it difficult to assess your standards just get a friend to help you be more objective.

And do not spend two hours making a spreadsheet to determine which friend would be optimal for the job.
Just don’t.

So you've probably established that your standards are out of control.
Or at the very least you're more open to the possibility that they're not really required for the Earth to keep spinning.

What do you do now?

## 2. Challenge Your Perfectionist Thoughts

From the book ["When Perfect isn't Good Enough"](https://www.amazon.com/When-Perfect-Isnt-Good-Enough/dp/157224559X):

> "An excellent way to test the accuracy of your perfectionistic thoughts and predictions is to carry out small experiments, a process also known as hypothesis testing."

That is you _**strategically half-ass it**_ and test your hypothesis that the Earth is going to melt as a consequence.

You show up five minutes late for that meeting.

You press send on that email without proofreading it twice.

And then wait...

So did the Earth actually explode?

And even if the result was "negative", was it as bad as you thought?

**But how much did you gain in terms of time, effort, and not driving yourself — or others — crazy?**

_Could less actually be more?_

Cognitive Behavioral Therapy – the most scientifically validated form of therapy out there – says we need to identify and RATIONALLY challenge our thoughts to see WHY we’re getting so bothered.

Because most of our problems with the world are caused by **irrational thoughts**.

Expose yourself to that thought and the consequence of not following through with your compulsive behavior.

Face your fears.

**HALF-ASS IT!**

If you expose yourself to what makes you uncomfortable frequently enough, it will become just as bland as watching the same movie 300 times.

Solid results will require some consistent dedicated practice.

But from personal experience, I can confirm this is the _most efficient way_ we know to overcome perfectionism.

## BONUS: the 70% rule

> Aim for 70%, not 100%.
> 
> And as soon as you get to ~70%, submit it and mark it as done.

I first heard about this from Youtuber Campbell Walker (on his channel [struthless](https://www.youtube.com/c/struthless)).

Actually, just watch his video below! You won't regret it.

https://www.youtube.com/watch?v=SxA69uUGEUI

> TLDR: here's how to overcome perfectionism
> 1. Reevaluate your standards
> 2. Challenge your perfectionist thoughts with strategic half-assing

This post is definitely over 70%, so I'm just gonna stop and ship it now 🚢 
Perfectionism be damned!

Now go do your magic.
